src/audio/data.h

```diff
@@ -148,14 +179,15 @@ extern OSMesgQueue *D_SH_80350FA8;
 #endif
 
 #if defined(VERSION_EU) || defined(VERSION_SH)
-#define AUDIO_HEAP_BASE 0x36B00
 #define AUDIO_INIT_POOL_SIZE (0x2C00 + EXT_AUDIO_INIT_POOL_SIZE)
 #else
-#define AUDIO_HEAP_BASE 0x30750
 #define AUDIO_INIT_POOL_SIZE (0x2500 + EXT_AUDIO_INIT_POOL_SIZE)
 #endif
 
-#define AUDIO_HEAP_SIZE (AUDIO_HEAP_BASE + EXT_AUDIO_HEAP_SIZE + EXT_AUDIO_INIT_POOL_SIZE + BETTER_REVERB_SIZE + REVERB_WINDOW_HEAP_SIZE)
+// TODO: needs validation once EU can compile. EU is very likely incorrect!
+#define AUDIO_HEAP_BASE (0x14D80 /* sound bank space */ + AUDIO_INIT_POOL_SIZE + EXT_AUDIO_HEAP_SIZE + NOTES_BUFFER_SIZE)
```

Comment:
i see `0x14D80` a few times, can you change this magic number to a define?

Reply:
Sure; If I change this though it will need to pretty much either replace EXPAND_AUDIO_HEAP completely or establish even more audio defines (which is why I had initially abstained).

Reply:
oh hmm

Reply:
i think a few defines is good, and suggest not touching those ones